python-squid/src/test/java/org/sonar/python/parser/python_v3/PythonV3Test.java

```diff
@@ -39,6 +39,7 @@ public void function_declaration(){
     setRootRule(PythonGrammar.FUNCDEF);
     assertThat(p).matches("def fun()->'Returns some value': pass");
     assertThat(p).matches("def fun(count:'Number of smth'=1, value:'Value of smth', type:Type): pass");
+    assertThat(p).matches("async def fun(): pass");
```

Comment:
@ivandalbosco Could you create a separate test method testing added syntax? Like that it's more visible which syntax is new and what are supported constructions